packages/framework-provider-azure-infrastructure/src/infrastructure/templates/cdktf.ts

```diff
@@ -0,0 +1,11 @@
+export const template = `{
+    "language": "typescript",
+    "app": "npm run --silent compile",
+    "projectId": "c13b5f64-7c69-4e12-93f9-a6daecd4f385",
```

Comment:
Shouldn't the projectId be dynamic?

Reply:
I guess we don't need this parameter. Let me check it and remove it if possible